interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -667,6 +692,9 @@ namespace cling {
                         IgnoreFilesFunc_t ignoreFiles =
                           [](const clang::PresumedLoc&) { return false;}) const;
 
+    void Complete(const std::string& line, size_t& cursor,
```

Comment:
That declaration is not used actually, with the renaming I forgot to delete it. Thank you!